packages/mask/src/social-network-adaptor/twitter.com/utils/selector.ts

```diff
@@ -210,7 +210,7 @@ export const postsContentSelector = () =>
 
 export const postAvatarsContentSelector = () => querySelectorAll('[data-testid="tweet"] > div > div  > :nth-child(2)')
 
-export const postAvatarSelector = () => querySelectorAll('[data-testid^=UserAvatar-Container-]')
+export const postAvatarSelector = () => querySelectorAll('[data-testid=tweet] [data-testid^=UserAvatar-Container-]')
```

Comment:
CC @UncleBill

Reply:
@lelenei This change would make it select Avatar Container in quoted tweets.

Reply:
<img width="665" alt="image" src="https://user-images.githubusercontent.com/72531217/223592870-35116475-0b15-4fde-8d53-7cbfea7a78ee.png">